### DISCUSSION TITLE
Problème de configuration VPN n'est pas lancé : et wan signalés avec une croix rouge

### DISCUSSION THREAD

**Discussion by infosb** [Ysurac/openmptcprouter#3396] [General]
Depuis la dernière mise à jour j'ai des croix rouge au lieu du logo vert sur mes wans une erreur annoncant vpn non lancé 
je ne sais trop ou chercher ...
ça semble fonctionner quand meme car le débit est correcte ...

**Comment by Ysurac** [comment]
Il faudrait beaucoup plus de détails... Et plutot ouvrir une issues en complétant le template.

**Reply by infosb** [reply]
ok je vais faire celà correctement  désolé !
